compiler/rustc_resolve/src/macros.rs

```diff
@@ -874,3 +903,14 @@ impl<'a> Resolver<'a> {
         result
     }
 }
+
+/// Convert the given number into the corresponding ordinal
+fn ordinalize(v: usize) -> String {
```

Comment:
Could you move this function to the `diagnostics.rs` module together with the tests for it?